packages/api/lib/writeRecords/write-granules.js

```diff
@@ -573,15 +630,19 @@ const _writeGranuleRecords = async (params) => {
  * Write a granule record to DynamoDB and PostgreSQL
  *
  * @param {Object}          params
+ * @param {Object}          params.postgresGranuleRecord - PostgreSQL granule record to write
+ *                                                         to thedatabase
  * @param {Object}          params.apiGranuleRecord - Api Granule object to write to the database
- * @param {number}          params.executionCumulusId - Execution ID the granule was written from
  * @param {Object}          params.esClient - Elasticsearch client
+ * @param {number}          params.executionCumulusId - Execution ID the granule was written from
  * @param {Object}          params.granuleModel - Instance of DynamoDB granule model
  * @param {Object}          params.granulePgModel - @cumulus/db compatible granule module instance
  * @param {Knex}            params.knex - Knex object
- * @param {Object}          params.postgresGranuleRecord - PostgreSQL granule record to write
- *                                                         to database
  * @param {string}          params.snsEventType - SNS Event Type
+ * @param {boolean}         params.writeConstraints - Boolean flag to set if createdAt/execution
```

Comment:
The typings on this should be updated, however this might also be a scope question given the logic in this method wasn't changed, and the edit here is a missing param in the jsdoc.

Reply:
Have not been able to figure this error out - `Qualified name 'params.xxxx' is not allowed without a leading '@param {object} params'`, but all the parameters above knex seem fine, only the bottom three are effected. They are also flagged in the parameter list, saying the property doesn't exist on the type. Will continue looking, but suggestions on how to fix appreciated.

Reply:
fixed, but in a weird way.